src/components/10-atoms/text/react/demo.jsx

```diff
@@ -0,0 +1,21 @@
+/* global document */
+import { storiesOf } from '@storybook/html';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DemoTextReact from './DemoTextReact';
+import Readme from '../README.md';
+import Changelog from '../CHANGELOG.md';
+
+storiesOf('Components|Atoms/Text/React/Demos', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+    changelog: Changelog,
+  })
```

Comment:
You can disable knobs tab here.

Reply:
... via `knobs:{disabled: true}` - thanks for the tip! 

Done.